sample-apps/client/ts-quickstart/src/main.ts

```diff
@@ -73,9 +76,17 @@ call.join({ create: true }).then(async () => {
   }
 
   container.appendChild(renderVolumeControl(call));
+
+  closedCaptionManager = new ClosedCaptionManager(call);
+  container.appendChild(closedCaptionManager.renderToggleElement());
+
+  const captionContainer = document.getElementById('closed-captions');
+  captionContainer?.appendChild(closedCaptionManager.renderCaptionContainer());
 });
 
 window.addEventListener('beforeunload', () => {
+  // Make sure to remove your event listeners when you leave a call
+  closedCaptionManager?.cleanup();
```

Comment:
Probably makes little difference to do this `beforeunload`, but a good practice nonetheless.

Reply:
Yes, I only added there to remind people about cleaning up in case they have a different implementatio